internal/framework/provider/provider.go

```diff
@@ -17,6 +17,7 @@ import (
 	"github.com/cloudflare/terraform-provider-cloudflare/internal/framework/muxclient"
 	"github.com/cloudflare/terraform-provider-cloudflare/internal/framework/service/access_mutual_tls_hostname_settings"
 	"github.com/cloudflare/terraform-provider-cloudflare/internal/framework/service/api_token_permissions_groups"
+	cloud_connector_rules "github.com/cloudflare/terraform-provider-cloudflare/internal/framework/service/cloud_connector"
```

Comment:
any reason why we cannot use the full name here instead of aliasing it? looks like you've named the package as such anyway.

Reply:
Yeap, my bad, fixed it